roles/openshift_grafana/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Add a Graphana URL base variable here so the operator can change the location of the 3 Graphana templates without changing the ansible code.

Reply:
done